pgml-extension/src/orm/snapshot.rs

```diff
@@ -283,16 +279,18 @@ impl Column {
 
         let mut modes = Vec::new();
         statistics.distinct = 0; // necessary reset before array columns clobber
-        for &value in &data {
+        for (i, &value) in data.iter().enumerate() {
```

Comment:
Why would it suggest that? `i` is not used, is it?

Reply:
It's used on line 307 I believe when calculating `v`. Previously it was used as an explicit counter outside of the for loop. https://rust-lang.github.io/rust-clippy/master/index.html#/explicit_counter_loop

Reply:
That's funny, wonder how we missed that !